sample_config.yaml

```diff
@@ -8,4 +8,5 @@ ingress:
     cert-manager.io/cluster-issuer: "linode-issuer"
     nginx.ingress.kubernetes.io/proxy-connect-timeout: "3600"
     nginx.ingress.kubernetes.io/proxy-read-timeout: "3600"
-    nginx.ingress.kubernetes.io/proxy-send-timeout: "3600"
\ No newline at end of file
+    nginx.ingress.kubernetes.io/proxy-send-timeout: "3600"
+hopli_image: europe-west3-docker.pkg.dev/hoprassociation/docker-images/hopli:latest
```

Comment:
This does not seem correct.

Reply:
What does not seem to be correct?
The hopli_image attribute is stored in the ConfigMap. This file is only used for local debugging. When the operator will run in Kubernetes, it will take the values from the configMap.